docs/source/common_issues.rst

```diff
@@ -229,16 +229,20 @@ Mypy may depend on orjson by default in the future.
 Types of empty collections
 --------------------------
 
+Without the annotation mypy can't always figure out the
```

Comment:
Linguistically this has to be rephrased or go back below the other part ("what is 'the annotation'?", thinks the reader of this sentence. "What is `a`?")